package.json

```diff
@@ -35,25 +35,25 @@
     "byline": "^5.0.0",
     "dugite-no-gpl": "1.69.0",
     "find-git-exec": "^0.0.3",
-    "upath": "^1.0.0"
+    "upath": "^2.0.1"
   },
   "devDependencies": {
     "@types/chai": "^4.2.5",
-    "@types/fs-extra": "^4.0.2",
-    "@types/mocha": "^5.2.7",
-    "@types/node": "^7.0.22",
-    "@types/temp": "^0.8.34",
+    "@types/fs-extra": "^9.0.12",
+    "@types/mocha": "^9.0.0",
+    "@types/node": "^16.4.7",
+    "@types/temp": "^0.9.1",
     "chai": "^4.2.0",
-    "concurrently": "^5.0.0",
-    "cross-env": "^6.0.3",
-    "fs-extra": "^4.0.2",
-    "mocha": "^6.2.2",
-    "node-ssh": "^6.0.0",
+    "concurrently": "^6.2.0",
+    "cross-env": "^7.0.3",
+    "fs-extra": "^10.0.0",
+    "mocha": "^9.0.3",
+    "node-ssh": "^12.0.0",
```

Comment:
The testing failures are due to these dev dependencies. E.g. `mocha@^9` needs Node 12. Please revert the dev dependencies to the previous versions (or at least versions compatible with Node 10).

Reply:
please see my comments in #46 I don't see how downgrading mocha could possibly solve the fact that the default git branch name is `main` on new machines, while the tests hardcode `master`. This means tests will fail when run locally. I'd rather fix these 4 failing tests by addressing the underlying  issues, before pinning an outdated mocha version.

Reply:
I see, makes sense 👍